js/src/edit-block/components/bottom-notice.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { QuestionIcon } from './';
+
+/**
+ * The bottom notice section.
+ *
+ * @return {React.ReactElement} The bottom notice.
+ */
+const BottomNotice = () => {
```

Comment:
Here's what this looks like:
<img width="998" alt="thi-bima" src="https://user-images.githubusercontent.com/4063887/102920020-83d7df80-444f-11eb-91fe-6a0e019c520e.png">

...though it only display on the onboarding post.